src/pages/registration/contributors_tab/components/ContributorCard.tsx

```diff
@@ -216,38 +200,28 @@ export const ContributorCard = ({
           />
         </StyledRightAlignedWrapper>
         <StyledCorrespondingWrapper>
-          <StyledCorrespondingField name={`${baseFieldName}.${SpecificContributorFieldNames.CORRESPONDING}`}>
+          <Field name={`${baseFieldName}.${SpecificContributorFieldNames.CORRESPONDING}`}>
             {({ field }: FieldProps) => (
               <FormControlLabel
                 data-testid="author-corresponding-checkbox"
-                control={<Checkbox checked={!!field.value} color="default" {...field} />}
+                control={
+                  <Checkbox
+                    checked={!!field.value}
+                    color="default"
+                    {...field}
+                    // TODO: Remove this block when backend has removed validation for email field
```

Comment:
finnes oppgave for dette? (i backend altså)

Reply:
nå gjør det det 👍  https://unit.atlassian.net/browse/NP-2282